react/QualificationModal/index.jsx

```diff
@@ -48,11 +48,11 @@ const QualificationModal = ({ file, title, onClose }) => {
   const client = useClient()
   const { t, lang } = useI18n()
 
-  const qualificationLabel = getQualification(file)?.label
+  const qualificationLabel = file.metadata?.qualification?.label
```

Comment:
Why not use the helper here?

Reply:
parce que le helper throw une erreur s'il y a une valeur manquante, alors qu'ici un undefined m’intéresse plus.